CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Change `REQUIRED_RUSK_VERSION` to `0.6.0` [#151]
 - Change `Error::Canon` variant to `Error::Rkyv` [#151]
 - Populate cache database with psk(s) on state init [#158]
+- Save the wallet.dat file with the new Rusk Binary Format and sha256 hashing
```

Comment:
- Missing the issue here
- Edit the message to adhere to the other entries (eg: "Change wallet file to support new RuskBinaryFormat")